src/main/java/org/folio/marccat/config/constants/Global.java

```diff
@@ -124,6 +124,8 @@
   public static final int AUT_TITLE_CATEGORY = 22;
   public static final int AUT_SUBJECT_CATEGORY = 18;
   public static final int AUT_NAME_TITLE_CATEOGRY = 11;
+  public static final int AUT_MATERIAL_DESCRIPTION_HEADER_TYPE = 10;
+  public static final String AUT_MATERIAL_DESCRIPTION_DESC = "008";
```

Comment:
Is `AUT` short for authority? If so, I'm not sure we save much by shortening it to `AUT`. I also think this Global file is getting a bit unwieldy. Perhaps we should consider breaking out domain constants into their own files, like Authority.java, with authority specific constants and move away from this Global file?

Also what does `AUT_MATERIAL_DESCRIPTION_DESC` mean? Is `DESC` short for description?

Finally, for my own understanding, what does the `10` in `AUT_MATERIAL_DESCRIPTION_HEADER_TYPE` mean?

Reply:
@carmentrazza, Do we split the file?

Yes, `AUT` means `AUTHORTIY` and `DESC` is `DESCRIPTION`. I have changed it.

`AUT_MATERIAL_DESCRIPTION_HEADER_TYPE` is a constant becouse we have in database this value to check that it is a 008 tag.

Reply:
Yes, that's fine

Reply:
We can move this properties in other PR.